### ISSUE TITLE
Add Δ between day price and trader price for item and per slot

### ISSUE THREAD

**Issue by marvk** [RatScanner/RatScanner#100]
**Is your feature request related to a problem? Please describe.**
Before unlocking the flea market, you are forced with decisions as to which barter items to keep and which to sell. Seeing at a glance how much profit you'd miss out on to make these decisions easier would, in my opinion, greatly enhance rat scanner.

**Describe the feature**
When checking the price of an item, add two more rows in the view:

* Δ price
* Δ price per Slot

The first one should show `average_day_price - trader_buy_price` and the second one should show `average_day_price_per_slot - trader_buy_price_per_slot`.

**Additional context**
I think it would be best if this feature were toggle-able, since the usefulness decreases dramatically as soon as you unlock the flea market.

**Comment by Blightbuster** [comment]
As far as I understand, this would require rat scanner to know, at what level your traders are at correct? @thaddeus Is it planned ( or even possible? ) to know this from the quest data provided by tarkovtracker?

**Comment by marvk** [comment]
@Blightbuster I think you must misunderstand. The terms of both calculations are already present as "Average Day Price", "Price per Slot" and "Buy Price". The only thing missing (but easily calculable) is "Buy Price per Slot".

**Comment by Blightbuster** [comment]
Oh so you want to know the difference between the market price and the price for which you could sell it to the trader?

**Comment by marvk** [comment]
Exactly, the point is to find the items with the minimal loss when selling when you're forced to sell when the inventory is filling up. It's of limited usefulness as it's only really relevant pre-flea, but it's probably easy enough to implement!

**Comment by Notare** [comment]
I support the idea, it would be very good to have a "price per slot" for the trader price as well, now that the Flea is lvl 20, I'm not even bothering with those options, I'm only using the "show trader price" option until I get to 20.

**Comment by marvk** [comment]
@Notare Yes, price per slot for trader would also be a good thing to have, though I'd prefer for those options to be toggle-able. I'm way past level 20 now and I would never be interested in them anymore.

**Comment by Notare** [comment]
> 
> 
> @Notare Yes, price per slot for trader would also be a good thing to have, though I'd prefer for those options to be toggle-able. I'm way past level 20 now and I would never be interested in them anymore.

Oh, I'm using minimal UI and all of the options are toggle-able, future options will probably be too.
